documentation/tenant/users.adoc

```diff
@@ -0,0 +1,65 @@
+= Users
+
+{ProductName} uses Red Hat Single Sign-On to manage users and credentials. You can create and delete {ProductName} users as well as change an existing password.
+
+[[_create_user]]
+== Creating a User
+
+Using the Red Hat Single Sign-On Administration Console you can create a new {ProductName} user and specify a password.
+
+.Procedure
+
+. In a web browser, navigate to http://localhost:8080/auth/admin/ and log in using the user name and password given to you for your realm.
```

Comment:
The URL here will not be correct in general, we should detail how to find the auth service admin console URL and connect to it from outside openshift for a given EnMasse installation